### PR TITLE
Formatting cleanup round 2

### PR DESCRIPTION
**Pull request by woodiertexas**
Formatting cleanup + qmj update + quilt deps update

### CODE REVIEW

File: src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockType.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public TypedGroupedBlock register(BlockGroup group, BlockGroup.GroupedBlock grou
 		var baseBlock = groupedBlock.baseBlock().get();
 
 		var blockSettings = QuiltBlockSettings.copyOf(baseBlock).strength(strength);
-		groupedBlock.mapColor().map(mapColor -> blockSettings.mapColor(mapColor));
+		groupedBlock.mapColor().map(mapColor -> blockSettings.mapColor(mapColor)); // replace lambda with method reference?
```
Comment (falkreon) [inline src/main/java/io/github/debuggyteam/architecture_extensions/api/BlockType.java]: either do it or don't. Don't leave the comment here, that's what IDE inspections are for.

File: src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java [hunk 2]
```diff
@@ -51,6 +51,7 @@ private static Cache loadOrGetCache(ModContainer resourceOwner, ResourceType res
 		});
 	}
 
+	// intelliJ says `static` is redundant for records
```
Comment (falkreon) [inline src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java]: remove this comment. We're not doing this.

**Review by falkreon — CHANGES_REQUESTED** [review]
There are still some dangling comments we need to resolve before merge.

File: src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java (hunk 2 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java]: > remove this comment. We're not doing this.

Yea I forgot to take this comment out yesterday.

**Review by falkreon — APPROVED** [review]

File: src/main/java/io/github/debuggyteam/architecture_extensions/DeferredRegistration.java [hunk 3]
```diff
@@ -87,7 +88,7 @@ public boolean register() {
 		
 		public Set<String> getIds() {
 			String modId = this.modId();
-			if (modId == "file") modId = ArchitectureExtensions.MOD_CONTAINER.metadata().id(); // If it's a staticdata resource, use our own id
+			if (Objects.equals(modId, "file")) modId = ArchitectureExtensions.MOD_CONTAINER.metadata().id(); // If it's a staticdata resource, use our own id
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/DeferredRegistration.java]:
```suggestion
			if (modId.equals("file")) modId = ArchitectureExtensions.MOD_CONTAINER.metadata().id(); // If it's a staticdata resource, use our own id
```

File: src/main/java/io/github/debuggyteam/architecture_extensions/DeferredRegistration.java [hunk 4]
```diff
@@ -3,6 +3,7 @@
 import java.util.Collection;
 import java.util.HashSet;
 import java.util.Iterator;
+import java.util.Objects;
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/DeferredRegistration.java]:
```suggestion
```

File: src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java [hunk 5]
```diff
@@ -69,12 +69,12 @@ public BlockGroup createBlockGroup() {
 	public Set<BlockType> getBlockTypes() {
 		Set<BlockType> result = new HashSet<>();
 		
-		for(String s : types_to_generate) {
-			BlockType blockType = BlockGroupSchema.<BlockType>reflectField(BlockType.class, s.toUpperCase(Locale.ROOT)).orElse(null);
+		for(String string : types_to_generate) {
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java]:
```suggestion
		for(String typeToGenerate : types_to_generate) {
```

File: src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java [hunk 6]
```diff
@@ -69,12 +69,12 @@ public BlockGroup createBlockGroup() {
...
+			BlockType blockType = BlockGroupSchema.<BlockType>reflectField(BlockType.class, string.toUpperCase(Locale.ROOT)).orElse(null);
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java]:
```suggestion
			BlockType blockType = BlockGroupSchema.<BlockType>reflectField(BlockType.class, typeToGenerate.toUpperCase(Locale.ROOT)).orElse(null);
```

File: src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java [hunk 7]
```diff
@@ -69,12 +69,12 @@ public BlockGroup createBlockGroup() {
...
 			if (blockType != null) {
 				result.add(blockType);
 			} else {
-				ArchitectureExtensions.LOGGER.warn("A file requested the nonexistant block type '"+s+"'.");
+				ArchitectureExtensions.LOGGER.warn("A file requested the nonexistant block type '" + string + "'.");
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/staticdata/BlockGroupSchema.java]:
```suggestion
				ArchitectureExtensions.LOGGER.warn("A file requested an nonexistent block type '" + typeToGenerate + "'.");
```

File: src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java [hunk 8]
```diff
@@ -50,7 +50,7 @@ private static Cache loadOrGetCache(ModContainer resourceOwner, ResourceType res
 			return null;
 		});
 	}
-
+
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java:53]: Why is there just floating whitespace here?

File: src/main/java/io/github/debuggyteam/architecture_extensions/ArchitectureExtensions.java [hunk 9]
```diff
@@ -68,7 +68,7 @@ public void onInitialize(ModContainer mod) {
 		for (EntrypointContainer<ArchExIntegration> entrypoint : QuiltLoader.getEntrypointContainers(ArchExIntegration.ENTRYPOINT_KEY, ArchExIntegration.class)) {
 			try {
 				entrypoint.getEntrypoint().integrate(new ArchExIntegrationContextImpl(entrypoint.getEntrypoint(), entrypoint.getProvider().metadata().id()));
-			} catch (Exception e) {
+			} catch (Exception ex) {
```
Comment (maximumpower55) [inline src/main/java/io/github/debuggyteam/architecture_extensions/ArchitectureExtensions.java:71]: I don't really like using `ex` for the exception, `e` was better imo.

File: .editorconfig [hunk 10]
```diff
@@ -4,31 +4,5 @@ root = true
 charset = utf-8
 end_of_line = lf
 insert_final_newline = true
-tab_width = 4
 trim_trailing_whitespace = false
```
Comment (maximumpower55) [inline .editorconfig:7]: trim trailing whitespace should be enabled

**Review by maximumpower55 — CHANGES_REQUESTED** [review]
Just a few nitpicks

File: src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java (hunk 8 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java:53]: editorconfig thing

File: .editorconfig (hunk 10 above)
Comment (falkreon) [reply .editorconfig:7]: trim trailing whitespace often results in messy commits as I intentionally leave trailing whitespace up to the indent of the block that contains it
e.g.
```java
public void foo() {
  //vertical whitespace here is indented at the block indent level
}
```

Comment (maximumpower55) [reply .editorconfig:7]: 👍

**Pull request merged**